src/vde_backend/utils.py

```diff
@@ -260,16 +259,16 @@ def energy2power(energy_df):
 	return power_df
```

Comment:
You did before energy_to_power and it is now energy2power. Do you have any preference or is it an oversight?

Reply:
It is because Guillaume said 'energy2power' actually respects the PEP specifications, thus I reverted the renaming.